docs/run-keep-ecdsa.adoc

```diff
@@ -0,0 +1,419 @@
+:toc: macro
+
+= Run ECDSA Keep
+
+:icons: font
+:numbered:
+toc::[]
+
+== System Considerations
+
+The Keep Network expects certain capabilites for each node running on the network.  To help attain
+these capabilities consider the following criteria:
+
+- It is paramount that Keep nodes remain available to the Keep Network. We strongly encourage a
+  stable and redundant internet connection.
+- A connection to a production grade self-hosted or third party Ethereum node deployment.
+- Persistent and redundant storage that will survive a VM or container rotation, and disk failure.
+- Each Keep ECDSA client running on the network requires a unique Ethereum operator account.
+- Each Keep ECDSA client running on the network requires a unique IP address or a unique
+  application port running under the same IP.
+- Recommended machine types by provider:
+
+[%header,cols=2*]
+|===
+|Cloud Provider
+|Machine Type
+
+|Google Cloud
+|n1-standard-4
+
+|AWS
+|m5.xlarge
```

Comment:
`c5.xlarge` is a 4x8 machine and costs about $18 less per month than the `m5.xlarge` as a suggestion to keep it cheaper on users.  I do not know the specs you are going off of, I am just using the specs of a self-hosted being 4x4 so my guess is that CPU is more important than memory.

Reply:
You are right that CPU is more important here.  My specs are based off more general application such as being a Kube Node, where having the available memory for other components is relevant.  We should probably capture both cases here, if you're running on a VM directly or as part of a multi-tenant setup such as Kube.

Reply:
I'm cheap mostly is why I brought this up.  I know a lot of people are cheap too so mostly trying to make it so that someone doesn't run a node because of the cost.